cloudferry_devlab/cloudferry_devlab/tests/README.md

```diff
@@ -0,0 +1,44 @@
+Directory with tests for Cloudferry's devlab
+==========
+
```

Comment:
I'd recommend adding general overview of how tests are run, that is:

1. Generate load in source cloud
2. Run migration
3. Verify migration results using nose and tests written